ccmlib/cmds/node_cmds.py

```diff
@@ -459,6 +460,41 @@ def run(self):
                                    column_families=self.column_families, size=self.size,
                                    no_snapshot=self.no_snapshot)
 
+class NodeGetsstablesCmd(Cmd):
+    def description(self):
+        return "Run getsstables to get absolute path of sstables in this node"
+
+    def get_parser(self):
+        usage = "usage: ccm node_name getsstables [options] [file]"
+        parser = self._get_default_parser(usage, self.description())
+        parser.add_option('-k', '--keyspace', type="string", dest="keyspace", default=None,
+                          help="The keyspace to use [use all keyspaces by default]")
+        parser.add_option('-c', '--tables', type="string", dest='tables', default=None,
```

Comment:
Can you make this short option `-t`? It doesn't make sense to have `-c` with `--tables`